root/label/edit_form.tt

```diff
@@ -62,10 +62,60 @@
   </div>
 
   <div class="documentation">
-    [%- area_bubble() -%]
+    <div class="bubble" id="name-bubble">
+      <p>
+        [% l('The {doc|name} is the label’s official name.',
+             { doc => { href => doc_link('Label#Name'), target => '_blank' } }) %]
+      </p>
+    </div>
+
+    <div class="bubble" id="comment-bubble">
+      <p>
+        [% l('The {doc|disambiguation} field is used to help users distinguish between identically‐ or similarly‐named labels.
```

Comment:
This (and later, the similar string in the place form) is basically the same as the artist form's string but `s/artist/label/g`. I couldn't think of a good way to avoid that without using "entity", though.

Reply:
That's fine IMO. Translators can reuse most of the string and only make a small change anyway.

Reply:
I agree that this is less of an issue for labels and places. Maybe something like this (keeping the second paragraph about annotations)?

> The disambiguation field is used to help users distinguish between labels with the same name. Feel free to leave this blank if you aren't warned about possible duplicates.